docs/content/tools/collector/_index.md

```diff
@@ -0,0 +1,494 @@
+```
+
+- Our next example will demonstrate how we can filter using a `NOT` operator. This will return all resources in scope, except those that meet the requirements of `app` or `application` not equalling `App3`, `env` or `environment` not equalling `dev` or `qa`.
+
+In the configFile.txt the configuration looks like:
+
+```text
+[tags]
+App||Application!~App3
+env||environment!~dev||qa
+```
+
+In PowerShell command line the configuration looks like:
+
+```powershell
+-tags "App||Application!~App3","env||environment!~dev||qa""
+```
+
+## Runbooks
+
+> __Important__: Runbooks are an advanced feature designed for specific workload-aligned use cases. If you're not sure if you need runbooks, you probably don't. Before diving into runbooks, [try using the filtering feature to see if it meets your needs](#resource-filtering).
+
+Learn more about using runbooks with the WARA collector script in the [runbooks docs](runbooks.md).
+
+## How to run the 1_wara_collector.ps1
+
+### Example 1
+
+The workload is entirely hosted in a single subscription that is not shared with other workloads:
+
+Provide parameters in command line:
+
+```powershell
+.\1_wara_collector.ps1 -TenantID "00000000-0000-0000-0000-000000000000" -SubscriptionIds "/subscriptions/00000000-0000-0000-0000-000000000000"
+```
+
+Provide parameters via ConfigFile:
+
+```powershell
+.\Wara_Collector.ps1 -configFile config.txt
+```
+
+config.txt
+
+```text
+[tenantId]
+00000000-0000-0000-0000-000000000000
+
+[subscriptionIds]
+/subscriptions/00000000-0000-0000-0000-000000000000
+```
+
+### Example 2
+
+The workload is hosted in two subscriptions (Sub-0 and Sub-5), but Sub 5 subscription is shared with other workloads and I want to filter the Sub-5 subscription by a single ResourceGroup where the components of my workload are hosted:
+
+Provide parameters in command line:
+
+```powershell
+.\1_wara_collector.ps1 -TenantID "00000000-0000-0000-0000-000000000000" -SubscriptionIds "/subscriptions/00000000-0000-0000-0000-000000000000" -ResourceGroups "/subscriptions/55555555-5555-5555-5555-555555555555/resourceGroups/RG-1"
+```
+
+Provide parameters via ConfigFile:
+
+```powershell
+.\Wara_Collector.ps1 -configFile config.txt
+```
+
+config.txt
+
+```text
+[tenantId]
+00000000-0000-0000-0000-000000000000
+
+[subscriptionIds]
+/subscriptions/00000000-0000-0000-0000-000000000000
+
+[resourceGroups]
+/subscriptions/55555555-5555-5555-5555-555555555555/resourceGroups/RG-1
+```
+
+**Important**: Note that since you are filtering Sub-5 by ResourceGroup, you do not pass Sub-5 in the Subscriptions parameter.
+
+### Example 3
+
+The workload is hosted in three subscriptions (Sub-0, Sub-3 and Sub-5), but Sub 5 subscription is shared with other workloads and I want to not only filter the Sub-5 subscription by a single ResourceGroup where the components of my workload are hosted but also I need to filter by two Tags called Criticality and Env, since all my subscriptions are hosting NonProduction resources too:
```

Comment:
```suggestion
The workload is hosted in three subscriptions (Sub-0, Sub-3, and Sub-5). However, since the Sub-5 subscription is shared with other workloads, I need to filter it not only by a specific Resource Group where the components of my workload are hosted but also by two tags, Criticality and Env, as all my subscriptions are hosting non-production resources as well.
```